### PR TITLE
Refresh the Office 365 documentation

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 1]
```diff
@@ -5,47 +5,71 @@ title: Office 365 Management
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Vue d'ensemble
+## Contenu du pack
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:8]:
```suggestion
Office 365 est une suite de services en ligne proposés par Microsoft dans le cadre de sa ligne de produit Microsoft Office.
Les informations de monitoring de la suite Office sont mises à disposition par Microsoft à travers une API de gestion Office 365.

## Contenu du pack
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 2]
```diff
@@ -5,48 +5,70 @@ title: Office 365 Management
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Overview
+## Pack assets
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:8]:
```suggestion
Office 365 is a line of online subscription services offered by Microsoft in their Microsoft Office product suite. 
Office 365 covers document creation and management, emailing, video conferencing and many more collaboration offerings.
The Centreon Plugin relies on the Office 365 Graph API to collect and monitor the Office 365 information and metrics.

## Pack assets
```

**Review by omercier — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 3]
```diff
@@ -5,47 +5,74 @@ title: Office 365 Management
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Vue d'ensemble
-
 Office 365 est une suite de services en ligne proposés par Microsoft dans le cadre de sa ligne de produit Microsoft Office.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:8]:
```suggestion
Office 365 est une suite de services en ligne proposés par Microsoft dans le cadre de sa ligne de produits Microsoft Office.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 4]
```diff
@@ -5,47 +5,74 @@ title: Office 365 Management
...
+| Service-Status | Cloud-Microsoft-Office365-Management-Service-Status-Api-custom | Contrôle le statut des services |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Cloud-Microsoft-Office365-Management-Api-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias           | Modèle de service                                               | Description                               |
+|:----------------|:----------------------------------------------------------------|:------------------------------------------|
+| App-Credentials | Cloud-Microsoft-Office365-Management-App-Credentials-Api-custom | Contrôle les credentials des applications |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:36]:
```suggestion
| App-Credentials | Cloud-Microsoft-Office365-Management-App-Credentials-Api-custom | Contrôle les informations de connexion des applications |
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 5]
```diff
@@ -5,48 +5,74 @@ title: Office 365 Management
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Overview
-
 Office 365 is a line of online subscription services offered by Microsoft in their Microsoft Office product suite. 
 Office 365 covers document creation and management, emailing, video conferencing and many more collaboration offerings.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:9]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 6]
```diff
@@ -135,116 +161,336 @@ when Azure AD redirects the tenant admin to your designated URL.
...
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_office365_management_api.pl \
+	--plugin=cloud::microsoft::office365::management::plugin \
+	--mode=subscriptions \
+	--tenant='' \
+	--client-id='' \
+	--client-secret=''  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:334]: fill in a sample value, as these are mandatory fields

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 7]
```diff
@@ -136,114 +163,336 @@ client_assertion=eyJhbGciOiJSUzI1NiIsIng1dCI6Ill5ZnNoSkMzclBRLWtwR281ZFVhaVk1dDN
...
-    --client-secret='9/kRTASjPoy9FJfQZg6iznX\AkzCGertBgNq5r3tPfECJfKxj6zA=' \
-    --filter-service-name='Exchange Online' \
-    --critical-status='%{status} !~ /serviceOperational|serviceRestored/i' \
-    --verbose
+/usr/lib/centreon/plugins/centreon_office365_management_api.pl \
+	--plugin=cloud::microsoft::office365::management::plugin \
+	--mode=subscriptions \
+	--tenant='' \
+	--client-id='' \
+	--client-secret=''  \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:336]: fill in a sample value, as these are mandatory fields

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 8]
```diff
@@ -136,114 +163,336 @@ client_assertion=eyJhbGciOiJSUzI1NiIsIng1dCI6Ill5ZnNoSkMzclBRLWtwR281ZFVhaVk1dDN
...
-    --client-secret='9/kRTASjPoy9FJfQZg6iznX\AkzCGertBgNq5r3tPfECJfKxj6zA=' \
-    --filter-service-name='Exchange Online' \
-    --critical-status='%{status} !~ /serviceOperational|serviceRestored/i' \
-    --verbose
+/usr/lib/centreon/plugins/centreon_office365_management_api.pl \
+	--plugin=cloud::microsoft::office365::management::plugin \
+	--mode=subscriptions \
+	--tenant='b3dd28de-012x2xxe-4d741212-bxx9-f093c1x9xx01' \
+	--client-id='28f82731-073b-4eb2-0012-001d252d2cb6-1b0d' \
+	--client-secret='9/kRTASjPxx9FJfQZg6iznX\AkzCGertBgNq5r3tPfECJfKxj6zA='  \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:336]:
```suggestion
	--tenant='xxxxxxxx-xxxxxxxx-xxxxxxxx-xxxx-xxxxxxxxxxxx' \
	--client-id='xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx-xxxx' \
	--client-secret='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'  \
```

File: pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md [hunk 9]
```diff
@@ -135,116 +161,336 @@ when Azure AD redirects the tenant admin to your designated URL.
...
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_office365_management_api.pl \
+	--plugin=cloud::microsoft::office365::management::plugin \
+	--mode=subscriptions \
+	--tenant='b3dd28de-012x2xxe-4d741212-bxx9-f093c1x9xx01' \
+	--client-id='28f82731-073b-4eb2-0012-001d252d2cb6-1b0d' \
+	--client-secret='9/kRTASjPxx9FJfQZg6iznX\AkzCGertBgNq5r3tPfECJfKxj6zA='  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-microsoft-office365-management.md:334]:
```suggestion
	--tenant='xxxxxxxx-xxxxxxxx-xxxxxxxx-xxxx-xxxxxxxxxxxx' \
	--client-id='xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx-xxxx' \
	--client-secret='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'  \
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**